static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -92,7 +91,11 @@ const MutationListItem = styled('li')`
 
 const MutationItemContainer = styled('div')`
   display: grid;
-  grid-template-columns: 280px 1fr;
+  grid-template-columns: 1fr;
+
+  @media (min-width: ${p => p.theme.breakpoints.large}) {
+    grid-template-columns: 280px 1fr;
+  }
```

Comment:
This is going to resize the content based on the size of the window. We need to resize things based on the size of the Tab area.

the window might be wide, but the area for the tab is narrow
it gets squished and is hard to use.